wallet/wallet.c

```diff
@@ -1297,7 +1297,8 @@ bool wallet_invoices_load(struct wallet *wallet, struct invoices *invs)
 	int count = 0;
 	sqlite3_stmt *stmt = db_query(__func__, wallet->db,
 				"SELECT id, state, payment_key, payment_hash, "
-				"label, msatoshi, expiry_time FROM invoices;");
+				"label, msatoshi, expiry_time, COALESCE(pay_index,0) "
```

Comment:
COALESCE() is unnecessary: the `inv->pay_index = sqlite3_column_int64(stmt, 7);` above will give 0 as you expect if the field is nul (I just re-checked the docs).